client/header/activity-panel/index.js

```diff
@@ -338,6 +383,22 @@ export class ActivityPanel extends Component {
 						{ this.renderPanel() }
 					</div>
 				</Section>
+				{ showHelpHighlightTooltip ? (
+					<HighlightTooltip
+						delay={ 1000 }
```

Comment:
I don't think this delay is working. As a test, I changed to `10000` and there was still no delay in the tooltip opening.

Reply:
Nice catch, I fixed it here: https://github.com/woocommerce/woocommerce-admin/pull/5826/commits/f0a38c741891924e8a21b32db6cb36eb02c8fa06